codalab/model/worker_model.py

```diff
@@ -49,6 +51,7 @@ def worker_checkin(self, user_id, worker_id, tag, cpus, gpus, memory_bytes, depe
                 'gpus': gpus,
                 'memory_bytes': memory_bytes,
                 'checkin_time': datetime.datetime.now(),
+                'available_disk_space': available_disk_space,
```

Comment:
Move this line to right after `memory_bytes` everywhere to try to keep the parallel structure